runtime/ftplugin/objdump.vim

```diff
@@ -0,0 +1,18 @@
+" Vim filetype plugin file
+" Language:     Objdump
+" Maintainer:   Colin Kennedy <colinvfx@gmail.com>
+" Last Change:  2023 October 25
+
+if exists("b:did_ftplugin")
+  finish
+endif
+
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
This `cpo` save/restore dance is only needed if using line continuations.  So lines 10,11,17,18 can be deleted.

Reply:
Ah okay, yes there are no line continuations in objdump's grammar. It's a line-by-line format. Done!

Reply:
It's not about objdump's grammar, it's more about whether you use line-continuation in this Vim script :)